### PR TITLE
refactor(bridge/nd): licenses & groups

### PR DESCRIPTION
**Pull request by Andyyy7666**

### CODE REVIEW

File: modules/bridge/nd/client.lua [hunk 1]
```diff
@@ -1,7 +1,41 @@
-RegisterNetEvent("ND:setCharacter", function(character)
-    local groups = {}
-    for group, info in pairs(character.groups) do
-        groups[group] = info.lvl
+function reorderGroups(groups)
```
Comment (thelindat) [inline modules/bridge/nd/client.lua:1]: `local function`

File: modules/bridge/nd/client.lua [hunk 2]
```diff
@@ -1,7 +1,41 @@
...
+end)
+
+RegisterNetEvent("ND:jobChanged", function(job, lastJob)
+    local character = NDCore.Functions.GetSelectedCharacter()
+    local groups = reorderGroups(character.data.groups)
+    groups[lastJob.name] = nil
+    groups[job.name] = job.rank
+    
+    OnPlayerData("groups", groups)
+end)
```
Comment (thelindat) [inline modules/bridge/nd/client.lua:31]: You have multi-groups but only update "job", and remove lastJob?

File: modules/bridge/nd/server.lua [hunk 3]
```diff
@@ -8,6 +8,13 @@ end)
 
 AddEventHandler("ND:characterUnloaded", playerDropped)
 
+RegisterNetEvent("ND:jobChanged", function(source, job, lastJob)
```
Comment (thelindat) [inline modules/bridge/nd/server.lua:11]: No "reorderGroups" for server?

File: modules/bridge/nd/server.lua [hunk 4]
```diff
@@ -82,3 +99,26 @@ function server.setPlayerData(player)
...
+function server.buyLicense(inv, license)
+	if server.hasLicense(inv, license.name) then
+		return false, "has_weapon_license"
+	elseif Inventory.GetItem(inv, "money", false, true) < license.price then
+		return false, "poor_weapon_license"
+	end
+
+	Inventory.RemoveItem(inv, "money", license.price)
+	NDCore.Functions.CreatePlayerLicense(inv.owner, "weapon")
+	return true, "bought_weapon_license"
```
Comment (thelindat) [inline modules/bridge/nd/server.lua:123]: Outdated locales?

**Review by thelindat — CHANGES_REQUESTED** [review]

File: modules/bridge/nd/client.lua (hunk 2 above)
Comment (Andyyy7666) [reply modules/bridge/nd/client.lua:31]: when job is set/changed then it will also set a group as that job

**Pull request merged**